taxonomy-editor-frontend/src/pages/search/results.jsx

```diff
@@ -0,0 +1,231 @@
+            >
+                <Typography sx={{mt: 2}} variant='h5'>{errorMessage}</Typography>
+                </Grid>   
+            </Container>
+        )
+    }
+
+    // Loading...
+    if (isPending) {
+        return (
+            <Container component="main" maxWidth="xs">
+            <Grid
+            container
+            direction="column"
+            alignItems="center"
+            justifyContent="center"
+            >
+                <Typography sx={{mt: 2}} variant='h5'>Loading..</Typography>
+                </Grid>   
+            </Container>
+        )
+    }
+
+    return (
+        <Box>
+            <Container component="main" maxWidth="xs">
+            <Grid
+            container
+            direction="column"
+            alignItems="center"
+            justifyContent="center"
+            >
+                <Grid item xs={3} sx={{mt: 4}}>
+                     <Typography variant="h4">Search Results</Typography>
+                </Grid>
+                <Typography variant="h6" sx={{mt: 2, mb: 1}}>
+                Number of nodes found: {nodes.length}
+                </Typography>
+                {/* Table for listing all nodes in taxonomy */}
+                <TableContainer sx={{width: 375}} component={Paper}>
+                    <Table>
+                        <TableHead>
+                        <TableRow>
+                            <Stack direction="row" alignItems="center">
+                                <TableCell align="left">
+                                <Typography variant="h6">
+                                    Nodes
+                                </Typography>
+                                </TableCell>
+                                <IconButton sx={{ml: 1, color: "#808080"}} onClick={handleOpenAddDialog}>
+                                    <AddBoxIcon />
+                                </IconButton>
+                            </Stack>
+                            <TableCell align="left">
+                            <Typography variant="h6">
+                                Action
+                            </Typography>
+                            </TableCell>
+                        </TableRow>
+                        </TableHead>
+                        <TableBody>
+                            {nodes.map((nodeID) => (
+                                <TableRow
+                                key={nodeID}
+                                >
+                                    <TableCell align="left" component="td" scope="row">
+                                        <Typography variant="subtitle1">
+                                            {nodeID}
+                                        </Typography>
+                                    </TableCell>
+                                    <TableCell align="left" component="td" scope="row">
+                                        <IconButton onClick={event => handleClick(event, nodeID) } aria-label="edit">
```

Comment:
could we use a `Link` instead?
also, please avoid generic names as `handleClick`

Reply:
Added `Link` type to the IconButton.